admin/commands/inventory/add_agent_external_exporter.go

```diff
@@ -49,15 +49,16 @@ func (res *addAgentExternalExporterResult) String() string {
 
 // AddAgentExternalExporterCommand is used by Kong for CLI flags and commands.
 type AddAgentExternalExporterCommand struct {
-	RunsOnNodeID string            `required:"" help:"Node identifier where this instance runs"`
-	ServiceID    string            `required:"" help:"Service identifier"`
-	Username     string            `help:"HTTP Basic auth username for scraping metrics"`
-	Password     string            `help:"HTTP Basic auth password for scraping metrics"`
-	Scheme       string            `help:"Scheme to generate URI to exporter metrics endpoints (http, https)"`
-	MetricsPath  string            `help:"Path under which metrics are exposed, used to generate URI"`
-	ListenPort   int64             `required:"" placeholder:"port" help:"Listen port for scraping metrics"`
-	CustomLabels map[string]string `mapsep:"," help:"Custom user-assigned labels"`
-	PushMetrics  bool              `help:"Enables push metrics model flow, it will be sent to the server by an agent"`
+	RunsOnNodeID  string            `required:"" help:"Node identifier where this instance runs"`
+	ServiceID     string            `required:"" help:"Service identifier"`
+	Username      string            `help:"HTTP Basic auth username for scraping metrics"`
+	Password      string            `help:"HTTP Basic auth password for scraping metrics"`
+	Scheme        string            `help:"Scheme to generate URI to exporter metrics endpoints (http, https)"`
+	MetricsPath   string            `help:"Path under which metrics are exposed, used to generate URI"`
+	ListenPort    int64             `required:"" placeholder:"port" help:"Listen port for scraping metrics"`
+	CustomLabels  map[string]string `mapsep:"," help:"Custom user-assigned labels"`
+	PushMetrics   bool              `help:"Enables push metrics model flow, it will be sent to the server by an agent"`
+	TLSSkipVerify bool              `help:"Skip TLS certificates validation"`
```

Comment:
```suggestion
	TLSSkipVerify bool              `help:"Skip TLS certificate verification"`
```

Reply:
IMO, it's about one certificate, not multiple. Furthermore, it's more about *verifying* whether it's signed by a trusted Certificate Authority rather than *validating* it.